sql/sys_vars.cc

```diff
@@ -6112,8 +6119,8 @@ static Sys_var_set Sys_log_slow_verbosity(
     "log_slow_verbosity",
     "Choose how verbose the messages to your slow log will be. "
     "Multiple flags allowed in a comma-separated string. [microtime, "
-    "query_plan, innodb, profiling, profiling_use_getrusage, minimal, "
-    "standard, full]",
+    "query_plan, innodb, profiling, profiling_use_getrusage, query_info, "
```

Comment:
can you please rename query_info to query_tables?

Reply:
I'm planning to use this flag for a few different fields in slow log so used a generic name for it.